### PR TITLE
CLDR-15034 kbd: update DTD, import should have distinguishing attrs

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15034

a few fixes:

- [X] update the DTD:  the attributes on `<import>` should be distinguishing, not 'value'.  Symptom: 'duplicate' errors from CLDR tests
- [x] update DTD with `@TECHPREVIEW` marker
- [x] update fr azerty so it builds. 
- …
- 

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
at some point i need to fork off this `fr` keyboard and just call it "TEST" or something, and then implement the real azerty, https://unicode-org.atlassian.net/browse/CLDR-12026

File: keyboards/3.0/fr-t-k0-azerty.xml [hunk 1]
```diff
@@ -52,19 +52,32 @@
...
+		<!-- switch keys -->
+		<key id="shift" switch="shift" />
+		<key id="numeric" switch="numeric" />
+		<key id="symbol" switch="symbol" />
+		<key id="base" switch="base" />
+
+
+		<key id="bksp" gap="true" /> <!-- TODO -->
+		<key id="extra" gap="true" /> <!-- TODO -->
+		<key id="enter" to="\u{000A}" /> <!-- TODO - implied key? -->
```
Comment (mcdurdin) [inline keyboards/3.0/fr-t-k0-azerty.xml:67]: The Enter key should not be generating a character -- that's an application-level decision. And certainly should not be defined in each keyboard; I can't see any need for Enter key to ever produce anything keyboard-specific (unlike spacebar).

File: keyboards/3.0/fr-t-k0-azerty.xml [hunk 2]
```diff
@@ -52,19 +52,32 @@
...
+		<import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+
+		<!-- switch keys -->
+		<key id="shift" switch="shift" />
+		<key id="numeric" switch="numeric" />
+		<key id="symbol" switch="symbol" />
+		<key id="base" switch="base" />
+
+
+		<key id="bksp" gap="true" /> <!-- TODO -->
```
Comment (mcdurdin) [inline keyboards/3.0/fr-t-k0-azerty.xml:65]: Why is bksp being defined as a gap? I don't think we should be able to redefine bksp or enter.

**Comment by mcdurdin** [comment]
I note we have nothing in the spec at present about `bksp`, `enter`, or `extra` being predefined key identifiers. Something for discussion perhaps?

**Comment by srl295** [comment]
Already on the agenda

File: keyboards/3.0/fr-t-k0-azerty.xml (hunk 1 above)
Comment (srl295) [reply keyboards/3.0/fr-t-k0-azerty.xml:67]: il'l take these keys out for now

**Comment by srl295** [comment]
merging this so i can make progress on tests. Feel free to post review or add comments here
